lib/model/query/entities.js

```diff
@@ -156,11 +157,14 @@ const _createEntity = (dataset, entityData, submissionId, submissionDef, submiss
 };
 
 const _updateEntity = (dataset, entityData, submissionId, submissionDef, submissionDefId, event) => async ({ Audits, Entities }) => {
+  if (!(event.action === 'submission.create')) // only update on submission.create
```

Comment:
should this condition be in `_processSubmissionEvent` function?

Reply:
Because I moved these lines into `_createEntity`

```
if ((dataset.approvalRequired && event.details.reviewState !== 'approved') ||
      (!dataset.approvalRequired && event.action === 'submission.update'))
```

it felt reasonable to move this event action check into `_updateEntity`